packages/frontend/src/hooks/useFieldValues.ts

```diff
@@ -43,7 +43,7 @@ export const useFieldValues = (
     const [selectedFieldName, setSelectedFieldName] = useState<string | null>();
     const [debouncedSearch, setDebouncedSearch] = useState<string>(search);
     const [searches, setSearches] = useState(new Set<string>());
-    const [results, setResults] = useState(new Set(...initialData));
+    const [results, setResults] = useState(new Set([...initialData]));
```

Comment:
Silly question, is `[...initialData]` not the same as `initialData`

Reply:
I did not dig deep but in MDN they usually spread arrays like that 😂 

<img width="635" alt="CleanShot 2023-02-15 at 13 19 56@2x" src="https://user-images.githubusercontent.com/962095/218986002-b2d95829-79df-4ce7-9f64-7f45953cec1e.png">

Reply:
I think that's because in that example they are spreading a set into a list. In your case it's already a list `string[]` on line 37